examples/asr/notebooks/6_VAD_using_NeMo.ipynb

```diff
@@ -0,0 +1,991 @@
+    "4. Authorize by run python freesound_download.py --authorize and visit website and paste response code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python freesound_download.py --authorize"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "5. Feel free to change any arguments for freesound_download.py in **download_resample_freesound.sh** such as max_samples and max_filesize\n",
+    "6. Run `bash download_resample_freesound.sh <max number of samples you want> <download data directory> <resampled data directory> `"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!bash download_resample_freesound.sh 4000 ./freesound  ./freesound_resampled_background"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the google speech command dataset\n",
+    "   \n",
+    "We will be using the open source Google Speech Commands Dataset (we will use V2 of the dataset for the tutorial, but require very minor changes to support V2 dataset) as our speech data. Google Speech Commands Dataset  v2 will take roughly 6GB disk space. These scripts below will download the dataset and convert it to a format suitable for use with nemo_asr.\n",
+    "\n",
+    "**NOTE**: You may additionally pass a `--rebalance` flag at the end of the `process_vad_data.py` script to rebalance the class samples (without duplicate) in the manifest. If you do so, please remember to change train_dataset below"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/process_vad_data.py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "speech_data_root = './google_dataset_v2'\n",
+    "background_data_root = './freesound_resampled_background' # your <resampled freesound data directory>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python process_vad_data.py --speech_data_root={speech_data_root} --background_data_root={background_data_root} --log=False --rebalance=True\n",
+    "print(\"Dataset ready !\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the path to manifest files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_dataset = './manifest/balanced_background_training_manifest.json,./manifest/balanced_speech_training_manifest.json' \n",
```

Comment:
Point these manifest paths to the new src directory where the code and data will be stored.

Reply:
I would prefer to create a new folder called out_dir ('./manifest') to store manifests produced by process_vad_data.py. And keep process_vad_data.py out of src folder because it's ugly to run a subfolder python script in ipynb I believe. But if you know some approach to do things like **!python src/process_vad_data.py --....** please let me know

Reply:
Because I have `tmp = 'src'`.  I was wondering to do something using variable name tmp. But this looks reasonable. Thank you! Will update it.

Reply:
Oh you can do that as `!python "$<name of variable>"` without the <>